src/main/java/com/endava/internship/collections/Main.java

```diff
@@ -0,0 +1,121 @@
+package com.endava.internship.collections;
+
+import java.time.LocalDate;
+import java.util.*;
```

Comment:
As I see you didn't configure your IntelijIdea https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports